packages/radix-ui-themes/src/components/container.tsx

```diff
@@ -2,8 +2,14 @@ import * as React from 'react';
 import classNames from 'classnames';
 import { Slot } from '@radix-ui/react-slot';
 import { containerPropDefs } from './container.props.js';
-import { extractProps, getRoot } from '../helpers/index.js';
-import { deprecatedLayoutPropDefs, layoutPropDefs, marginPropDefs } from '../props/index.js';
+import { extractProps, getRoot, subtract } from '../helpers/index.js';
+import {
+  deprecatedLayoutPropDefs,
```

Comment:
Just checking, are we going to remove the deprecated stuff now that we're making a major version?

Reply:
Yep, just separately